### PR TITLE
Fix wrong count for replica's tot-net-out

### PR DESCRIPTION
**Pull request by uriyage**
Fix duplicate calculation of replica's `net_output_bytes`

- Remove redundant calculation leftover from previous refactor
- Add test to prevent regression

### CODE REVIEW

**Review by zuiderkwast — APPROVED** [review]
Looks correct to me. Thanks for the test.

You say the calculation was counted twice. Does that mean that without this fix, the test reports more than 20000 bytes for tot-net-out after replicating a 10000 bytes key?

**Review by enjoy-binbin — APPROVED** [review]
ok, so the old net_output_bytes is in line 2197 (or line 2190). LGTM.

**Comment by uriyage** [comment]
> Looks correct to me. Thanks for the test.
> 
> You say the calculation was counted twice. Does that mean that without this fix, the test reports more than 20000 bytes for tot-net-out after replicating a 10000 bytes key?

Yes.

File: tests/unit/introspection.tcl [hunk 1]
```diff
@@ -836,6 +836,43 @@ start_server {tags {"introspection"}} {
...
+                set info_list [$primary client list]
+                foreach info [split $info_list "\r\n"] {
+                    if {[string match "* flags=S *" $info]} {
+                        set out [get_field_in_client_info $info "tot-net-out"]
+                        break
+                    }
+                }
+
+                assert {$out > 0}
+                assert {$out < [expr $value_size + 1000]}
```
Comment (enjoy-binbin) [inline tests/unit/introspection.tcl]:
```suggestion
                assert_morethan $out 0
                assert_lessthan $out [expr $value_size + 1000]
```

**Review by ranshid — APPROVED** [review]
LGTM.

I think maybe we can straighten the test by disabling primary+replica KA and increase the repl-timeout so that this will not theoretically becomes flaky?

**Comment by uriyage** [comment]
> LGTM.
> 
> I think maybe we can straighten the test by disabling primary+replica KA and increase the repl-timeout so that this will not theoretically becomes flaky?

Thanks, Changed per your suggestion.

**Pull request merged**